app/src/main/java/com/nervousfish/nervousfish/activities/SettingsActivity.java

```diff
@@ -289,4 +285,72 @@ public boolean onOptionsItemSelected(final MenuItem item) {
             return super.onOptionsItemSelected(item);
         }
     }
+
+    private static final class BindPreferenceToValueListener implements Preference.OnPreferenceChangeListener {
+        @Override
+        public boolean onPreferenceChange(final Preference preference, final Object newValue) {
```

Comment:
You actually didn't remove it, but it was indeed not used so I removed it for you 🙂